src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square() {
+        this.side = new Random().nextInt(15);
```

Comment:
lets init values outside
exact FigureSupplier should generate figures with random properties:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/ce46f7a7-b894-46cb-8d3f-d171679c653c)
```suggestion
```